### ISSUE TITLE
Uncaught TypeError: Cannot read property 'destroyItem' of undefined

### ISSUE THREAD

**Issue by ricardooow** [lightbulb-softworks/atom-realtime-collaboration#1]
[Enter steps to reproduce:]

1. disconnect
2. ...

**Atom**: 1.18.0 x64
**Electron**: 1.3.15
**OS**: Mac OS X 10.12.5
**Thrown From**: [atom-realtime-collaboration](https://github.com/lightbulb-softworks/atom-realtime-collaboration) package 0.6.2

### Stack Trace

Uncaught TypeError: Cannot read property 'destroyItem' of undefined

```
At /Users/ricardo/.atom/packages/atom-realtime-collaboration/lib/RemoteConnection.js:185

TypeError: Cannot read property 'destroyItem' of undefined
    at RemoteConnection.closeTab (/packages/atom-realtime-collaboration/lib/RemoteConnection.js:185:15)
    at RemoteConnection.remove (/packages/atom-realtime-collaboration/lib/RemoteConnection.js:209:10)
    at /packages/atom-realtime-collaboration/lib/RemoteConnection.js:109:64
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at TextEditor.module.exports.TextEditor.destroyed (/Applications/Atom.app/Contents/Resources/app/src/text-editor.js:584:26)
    at TextEditor.module.exports.Model.destroy (/Applications/Atom.app/Contents/Resources/app/src/model.js:31:64)
    at Pane.module.exports.Pane.destroyItem (/Applications/Atom.app/Contents/Resources/app/src/pane.js:687:24)
    at TabBarView.module.exports.TabBarView.closeTab (/Applications/Atom.app/Contents/Resources/app/node_modules/tabs/lib/tab-bar-view.js:340:32)
    at HTMLDivElement.TabView.didClickCloseIcon (/Applications/Atom.app/Contents/Resources/app/node_modules/tabs/lib/tab-bar-view.js:228:25)
```

### Commands

```
     -0:46.4.0 editor:copy-path (input.hidden-input)
     -0:45.1.0 intentions:highlight (input.hidden-input)
     -0:44.7.0 core:copy (input.hidden-input)
     -0:42 atom-realtime-collaboration:connect (input.hidden-input)
 10x -0:40 core:backspace (input.hidden-input)
     -0:37.2.0 core:confirm (input.hidden-input)
  3x -0:22.7.0 editor:newline (input.hidden-input)
```

### Non-Core Packages

```
advanced-open-file 0.16.6 
atom-beautify 0.30.3 
atom-pair 2.0.10 
atom-realtime-collaboration 0.6.2 
autocomplete-paths 1.0.5 
busy-signal 1.4.3 
color-picker 2.2.5 
emmet 2.4.3 
git-plus 7.9.3 
gpp-compiler 3.0.7 
highlight-selected 0.13.1 
intentions 1.1.2 
linter 2.2.0 
linter-jshint 3.1.5 
linter-ui-default 1.6.2 
pigments 0.39.1 
seti-icons 1.5.4 
simple-drag-drop-text 0.3.4 
```

**Comment by andrycodestuffs** [comment]
Thanks for reporting this issue, the problem is now fixed in the 0.6.3 release

**Comment by andrycodestuffs** [comment]
Thanks for reporting this issue, the problem is now fixed in the 0.6.3 release
